spaceship/spec/client_spec.rb

```diff
@@ -100,4 +112,11 @@ def subject.send_login_request(user, password)
       end
     end
   end
+
+  describe "#log_response" do
+    it 'handles ASCII-8BIT to UTF-8 encoding gracefully' do
+      response = TestResponse.new([130, 5, 3120, 130, 4, 171, 160, 3, 2].pack('C*'))
+      expect(subject.try_log_response(:get, TestClient.hostname, response)).to be_truthy
```

Comment:
Oh, because log_response is a private method - plus I was keeping with the convention of how this class was setup.  Do you prefer a different way to invoke it?